app.rb

```diff
@@ -0,0 +1,65 @@
+require './student'
+require './teacher'
+require './book'
+require './classroom'
+
+class App
+  attr_reader :books, :classrooms, :students, :teachers
+
+  def initialize
+    @students = []
+    @books = []
+    @teachers = []
+    @rentals = []
+    @classrooms = []
+  end
+
+  def list_people
+    @students.each { |s| puts "[Student] ID: #{s.id}, Name: #{s.name}" }
+    @teachers.each { |t| puts "[Teacher] ID: #{t.id}, Name: #{t.name}" }
+  end
+
+  def list_books
+    @books.each { |b| puts "[Book] Title: #{b.title}, Author: #{b.author}" }
+  end
+
+  def list_classrooms
+    @classrooms.each { |c| puts "[Classroom] Label: #{c.label}" }
+  end
+
+  def create_student(age, name, classroom)
+    student = Student.new(age, name, classroom)
+    @students << student
+    student
+  end
+
+  def create_teacher(age, name, specialization)
+    teacher = Teacher.new(age, name, specialization)
+    @teachers << teacher
+    teacher
+  end
```

Comment:
- [ ] Please as mentioned by the previous reviewer, you should make sure that your app works correctly. When I try to create a person it's breaking the app, please kindly fix the error.
You can take a look at how the app breaks when I create a person 🙃 

https://user-images.githubusercontent.com/74501523/189474365-8dae4bc2-7385-4235-bc7f-9965f9d3239f.mov